spec/css-spec.coffee

```diff
@@ -95,89 +113,54 @@ describe 'CSS grammar', ->
 
-    describe 'custom elements', ->
-      it 'tokenizes them as tags', ->
-        {tokens} = grammar.tokenizeLine 'very-custom { color: red; }'
+    describe 'compound selectors', ->
+      it 'tokenizes type selector with class selector', ->
+        {tokens} = grammar.tokenizeLine 'very-custom.class'
         expect(tokens[0]).toEqual value: 'very-custom', scopes: ['source.css', 'meta.selector.css', 'entity.name.tag.custom.css']
+        expect(tokens[1]).toEqual value: '.', scopes: ['source.css', 'meta.selector.css', 'entity.other.attribute-name.class.css', 'punctuation.definition.entity.css']
+        expect(tokens[2]).toEqual value: 'class', scopes: ['source.css', 'meta.selector.css', 'entity.other.attribute-name.class.css']
 
-        {tokens} = grammar.tokenizeLine 'very-very-custom { color: red; }'
-        expect(tokens[0]).toEqual value: 'very-very-custom', scopes: ['source.css', 'meta.selector.css', 'entity.name.tag.custom.css']
-
-      it 'tokenizes them with pseudo selectors', ->
-        {tokens} = grammar.tokenizeLine 'very-custom:hover { color: red; }'
+      it 'tokenizes type selector with pseudo class', ->
+        {tokens} = grammar.tokenizeLine 'very-custom:hover'
         expect(tokens[0]).toEqual value: 'very-custom', scopes: ['source.css', 'meta.selector.css', 'entity.name.tag.custom.css']
         expect(tokens[1]).toEqual value: ':', scopes: ['source.css', 'meta.selector.css', 'entity.other.attribute-name.pseudo-class.css', 'punctuation.definition.entity.css']
         expect(tokens[2]).toEqual value: 'hover', scopes: ['source.css', 'meta.selector.css', 'entity.other.attribute-name.pseudo-class.css']
 
-        {tokens} = grammar.tokenizeLine 'very-custom::shadow { color: red; }'
+      it 'tokenizes type selector with pseudo element', ->
+        {tokens} = grammar.tokenizeLine 'very-custom::shadow'
         expect(tokens[0]).toEqual value: 'very-custom', scopes: ['source.css', 'meta.selector.css', 'entity.name.tag.custom.css']
         expect(tokens[1]).toEqual value: '::', scopes: ['source.css', 'meta.selector.css', 'entity.other.attribute-name.pseudo-class.css', 'punctuation.definition.entity.css']
         expect(tokens[2]).toEqual value: 'shadow', scopes: ['source.css', 'meta.selector.css', 'entity.other.attribute-name.pseudo-class.css']
 
-      it 'tokenizes them with class selectors', ->
-        {tokens} = grammar.tokenizeLine 'very-custom.class { color: red; }'
-        expect(tokens[0]).toEqual value: 'very-custom', scopes: ['source.css', 'meta.selector.css', 'entity.name.tag.custom.css']
-        expect(tokens[1]).toEqual value: '.', scopes: ['source.css', 'meta.selector.css', 'entity.other.attribute-name.class.css', 'punctuation.definition.entity.css']
-        expect(tokens[2]).toEqual value: 'class', scopes: ['source.css', 'meta.selector.css', 'entity.other.attribute-name.class.css']
-
-      it 'does not confuse them with properties', ->
```

Comment:
I guess it should be fine. I mean, look at the new pattern of custom tag, whether it is followed by spaces or colon is not really relevant. And I think this test is not quite necessary in the first place unless people try to include the `#selector` entry in `#rule-list` and put it before the pattern of properties, it won't be confused with the properties. Besides, the property lists suite already has a test for extraneous spaces. 😉